test/functional/Valhalla/playlist.xml

```diff
@@ -53,4 +53,37 @@
 			<impl>ibm</impl>
 		 </impls>
 	</test>
+	<test>
+		<testCaseName>ValueTypeTestsJIT</testCaseName>
+		<variations>
+			<!--
+			  - -Xjit:count=0 temporarily disabled due to intermittent failures
+			  -
+			  -	<variation>-Xjit:count=0</variation>
+			  -->
+			<variation>-Xjit:count=1,disableAsyncCompilation</variation>
+		</variations>
+		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) \
+		-Xverify:none \
+		--add-opens java.base/jdk.internal.misc=ALL-UNNAMED \
+		-XX:+EnableValhalla \
+		-cp $(Q)$(LIB_DIR)$(D)asm-all.jar$(P)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_RESROOT)$(D)ValhallaTests.jar$(Q) \
+		org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng.xml$(Q) -testnames ValueTypeTestsJIT \
+		-groups $(TEST_GROUP) \
+		-excludegroups $(DEFAULT_EXCLUDE); \
+		$(TEST_STATUS)</command>
+		<levels>
+			<level>sanity</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<subsets>
+			<subset>Valhalla</subset>
+		</subsets>
+		<impls>
+			<impl>openj9</impl>
+			<impl>ibm</impl>
+		 </impls>
```

Comment:
minor indentation issue here

Reply:
Fixed indentation issue in commit https://github.com/eclipse/openj9/commit/5e799b4cef13b70ed659ec0c153d12afbf1b5c6b